TurfRegressionTests/Common/UnindexedTreesCollection.swift

```diff
@@ -0,0 +1,45 @@
+import Foundation
+
+import Turf
+
+final class UnindexedTreesCollection: Collection {
```

Comment:
Yeah I think you'll find you can test the negative here. Testing the secondary index methods is enough for a regression test.